apps/build.groovy

```diff
@@ -94,10 +94,10 @@ EOF
 	}
 
 	return new AppBuildResults(
-		dbMigratorZip: 'apps/bfd-db-migrator/target/bfd-db-migrator-1.0.0-SNAPSHOT.zip',
-		dataPipelineZip: 'apps/bfd-pipeline/bfd-pipeline-app/target/bfd-pipeline-app-1.0.0-SNAPSHOT.zip',
-		dataServerLauncher: 'apps/bfd-server/bfd-server-launcher/target/bfd-server-launcher-1.0.0-SNAPSHOT.zip',
-		dataServerWar: 'apps/bfd-server/bfd-server-war/target/bfd-server-war-1.0.0-SNAPSHOT.war'
+		dbMigratorZip:      sh(returnStdout: true, script: """find "${workspace}/apps/bfd-db-migrator/target" -type f -name bfd-db-migrator-*.zip""").trim(),
```

Comment:
just curious why you searched for both 'type' and then 'name'...

Reply:
It's _probably_ not necessary here. At worst, the inclusion of `-type f` is harmless, but it is more explicit and potentially safer to limit the result set to files rather than say... directories?

There isn't a lot of safety in this process, but if a directory somehow came to be named e.g. `bfd-db-migrator-1.0.0-SNAPSHOT.zip` here and was filtered out as a consequence of the `type -f`, the packer build would fail due to an inaccessible _or_ non-existent path on the ansible host (at Jenkins) represented by an empty `db_migrator_zip` ansible variable. If a _directory_ were passed in without this filtering, packer+ansible build would receive this mysterious directory, but it might be less obvious and take longer to reach failure. The problem would be a little later on the EC2 instance targeted by packer+ansbile. Hmm.